src/main/java/com/devonfw/ide/sonarqube/common/api/config/ArchitectureJsonReader.java

```diff
@@ -0,0 +1,142 @@
+package com.devonfw.ide.sonarqube.common.api.config;
+
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+
+/**
+ *
+ */
+public class ArchitectureJsonReader {
```

Comment:
Please use/extend existing `ConfigurationMapper` instead of creating this new class:
https://github.com/devonfw/sonar-devon4j-plugin/blob/master/src/main/java/com/devonfw/ide/sonarqube/common/impl/config/ConfigurationMapper.java

Reply:
ok, sure